spec/models/abilities/sdg/manager.rb

```diff
@@ -9,6 +9,7 @@
 
   it { should be_able_to(:read, SDG::Goal) }
   it { should be_able_to(:read, SDG::Target) }
+  it { should be_able_to(:read, SDG::LocalTarget) }
```

Comment:
Maybe in this case permissions to create, update and destroy are more important :thinking:.
```suggestion
  it { should be_able_to(:manage, SDG::LocalTarget) }
```